contributor_docs/hi/contributor_guidelines.md

```diff
@@ -406,94 +406,105 @@ p5.js की किसी भी विशेषता पर काम कर
 
-Now that you have made the changes you need to make, including unit tests if applicable, `npm test` does not error, and you have committed the changes, you can start preparing pull requests to get your new commits merged into the official p5.js repository. A pull request, more formally, is a request to a repo (in this case, the official p5.js repo) to pull or merge changes from another repo (in this case, your forked p5.js repo) into its commit history.
+अब जब आपने आवश्यक परिवर्तन कर लिए हैं, यदि लागू हो तो इकाई परीक्षण भी शामिल है, `npm test` में गलती नहीं होती है, और आपने कमिट  कर दिए हैं, तो आप अपने नए कमिट को आधिकारिक p5.js में मर्ज करने के लिए पुल रिक्वेस्ट तैयार करना शुरू कर सकते हैं जो p5.js रिपॉजिटरी में मर्ज हो जाएगा। एक पुल रिक्वेस्ट, अधिक औपचारिक रूप से, एक रेपो (इस मामले में, आधिकारिक p5.js रेपो) के लिए एक अन्य रेपो (इस मामले में, आपके फोर्कड p5.js रेपो) से परिवर्तनों को उसके प्रतिबद्ध इतिहास में पुल या मर्ज करने का अनुरोध है।
 
-## Creating a pull request
 
-The first step here is to push your new commits to your fork of p5.js; think of it as uploading the changes to your fork.
+## पुल रिक्वेस्ट बनाना 
 
-From GitHub Desktop, just to the right of the button to change branches in the header is a button to push your changes to GitHub. Click this to push your changes.![A view of GitHub Desktop after committing changes. The button to push the changes online is circled in red.](../images/publish-branch.png)
+यहां पहला कदम आपके नए कमिट को आपके p5.js के फोर्क पर धकेलना है; इसे अपने फोर्क में परिवर्तन अपलोड करने के रूप में सोचें।
 
-Once your code is uploaded, it will show a button prompting you to create a pull request. Clicking the button once will show a preview with another button to actually create the request. Press the "Create Pull Request" button to begin the process.
 
-![A screenshot of Github Desktop after pushing code. In the left sidebar, it says "0 changed items." In the right pane, below the "No local changes" header, a blue "Review Pull Request" button has been marked up with a red circle.](../images/preview-pull-request.png)
+GitHub डेस्कटॉप से, हेडर में शाखाएँ बदलने के बटन के ठीक दाईं ओर आपके परिवर्तनों को GitHub पर भेजने के लिए एक बटन है। अपने परिवर्तनों को आगे बढ़ाने के लिए इस पर क्लिक करें।![परिवर्तन करने के बाद GitHub डेस्कटॉप का एक दृश्य। परिवर्तनों को ऑनलाइन पुश करने का बटन लाल रंग में दर्शाया गया है।](../images/publish-branch.png)
 
-From the terminal, run the following command:
+
+एक बार आपका कोड अपलोड हो जाने पर, यह एक बटन दिखाएगा जो आपको पुल रिक्वेस्ट बनाने के लिए प्रेरित करेगा। बटन पर एक बार क्लिक करने से वास्तव में रिक्वेस्ट बनाने के लिए दूसरे बटन के साथ पूर्वावलोकन दिखाई देगा। प्रक्रिया शुरू करने के लिए "क्रिएट पुल रिक्वेस्ट" बटन दबाएँ।
+
+![कोड पुश करने के बाद Github Desktop का स्क्रीनशॉट। बाएँ साइडबार में, यह लिखा है "0 changed items." दाएँ फलक में, "No local changes" शीर्षक के नीचे, एक नीला "Review Pull Request" बटन को लाल वृत्त के साथ चिह्नित किया गया है।](../images/preview-pull-request.png)
```

Comment:
Gtihub Desktop in hindi

Reply:
I think we can keep it as GitHub Desktop as it is name of the application which hey will read in English by default.